HopsanGUI/Dialogs/ComponentPropertiesDialog3.cpp

```diff
@@ -1804,9 +1804,14 @@ void ParameterValueSelectionWidget::openValueEditDialog()
 
 void ParameterValueSelectionWidget::openFileBrowserDialog()
 {
-    QString filePath = QFileDialog::getOpenFileName(gpMainWindowWidget, "Select file", gpDesktopHandler->getDocumentsPath());
+    QString defaultPath = gpDesktopHandler->getDocumentsPath();
+    if(!mpValueEdit->text().isEmpty()) {
+        defaultPath = QFileInfo(mpValueEdit->text()).absolutePath();
+    }
+    QString filePath = QFileDialog::getOpenFileName(gpMainWindowWidget, "Select file", defaultPath);
     if(!filePath.isEmpty()) {
-        mpValueEdit->setText(filePath);
+        QDir relDir(gpDesktopHandler->getExecPath());
```

Comment:
I think that it would be better to be relative to the model, if it is possible to get the model path here

Reply:
Ok, I changed it.

For some reason the model file info is stored in the top level system and not the model widget. I think we should move that, but not now.